config/routes.rb

```diff
@@ -4,4 +4,5 @@
 
   resources :games
```

Comment:
resources: games doesn't require any controllers currently, may want to limit resource generation